### PR TITLE
Fixed broken table, corrected incorrect undo assert about DDL

### PR DESCRIPTION
**Pull request by Denubis**
The table of datatypes was broken.

And most databases can't ROLLBACK a DDL statement, as DDL statements (by the SQL spec) include a COMMIT before they execute. Thus, I changed that line to "backup" rather than "undo"

Fixed capitalisation on VALUES for consistency.

### CODE REVIEW

**Comment by Denubis** [comment]
As an issue to raise here, I think there should be a COMMIT/ROLLBACK exercise at the bottom, but we are firmly enjoined not to create exercises. Let me know if I should write up a few lines about those commands.

**Comment by henrykironde** [comment]
The changes look ready we just need to make sure the table renders perfectly. Looks fine in markdown view.

**Comment by remram44** [comment]
The table fix is already in #166.

I think the statement about undoing is misleading. Databases don't have a way of undoing changes, apart from very few specialized systems. They all support rolling back transactions, but either you are in a transaction or you're not. Schema-altering statements also commit, as you rightly point out.

File: _episodes/09-create.md [hunk 1]
```diff
@@ -46,18 +46,18 @@ DROP TABLE Survey;
...
-INTEGER    a signed integer
-REAL       a floating point number
-TEXT       a character string
-BLOB       a "binary large object", such as an image
+data type|use
+---------|-----------------------------------------
+INTEGER  |  a signed integer
+REAL     |  a floating point number
+TEXT     |  a character string
+BLOB     |  a "binary large object", such as an image
```
Comment (remram44) [inline _episodes/09-create.md]: This is going to conflict after #166

**Pull request merged**